ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/gen_client_certs.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
temp file: to be used if you need/want to run tests against caikit-nlp-client library